addons/test_mail/tests/test_performance.py

```diff
@@ -271,7 +271,7 @@ def test_mail_composer(self):
                 'partner_ids': [(4, customer_id)],
             })
 
-        with self.assertQueryCount(__system__=30, emp=36):
+        with self.assertQueryCount(__system__=31, emp=37):
```

Comment:
Why do we still have extra queries? Normally `fetch_followers` should be `False` in all of those flows. Only actual fetch RPC from JS inbox should have it to `True`.

Reply:
This is still an open concern.